src/rgw/rgw_data_sync.cc

```diff
@@ -5168,6 +5192,7 @@ class RGWSyncBucketCR : public RGWCoroutine {
   rgw_bucket_index_marker_info info;
 
   RGWSyncTraceNodeRef tn;
+  BucketIndexShardsManager remote_markers;
```

Comment:
this doesn't appear to be used in `class RGWSyncBucketCR`

Reply:
Done!